src/status_im/navigation/screens.cljs

```diff
@@ -322,11 +316,9 @@
      :component syncing-results/view}
 
     {:name      :welcome
-     :options   (merge
```

Comment:
these are all unnecessary merges, we are already merging these options while navigating